package.json

```diff
@@ -18,11 +18,10 @@
     "coverage": "forge coverage --match-contract Unit",
     "deploy:mainnet": "bash -c 'source .env && forge script Deploy -vvvvv --rpc-url $MAINNET_RPC --broadcast --chain mainnet --private-key $MAINNET_DEPLOYER_PK --verify --etherscan-api-key $ETHERSCAN_API_KEY'",
     "deploy:testnet": "bash -c 'source .env && forge script Deploy -vvvvv --rpc-url $SEPOLIA_RPC --broadcast --chain sepolia --private-key $SEPOLIA_DEPLOYER_PK --verify --etherscan-api-key $ETHERSCAN_API_KEY'",
-    "lint:check": "yarn lint:sol-tests && yarn lint:sol-logic && forge fmt --check",
-    "lint:fix": "sort-package-json && forge fmt && yarn lint:sol-tests --fix && yarn lint:sol-logic --fix",
+    "lint:check": "yarn lint:sol && forge fmt --check",
+    "lint:fix": "sort-package-json && forge fmt && yarn lint:sol --fix",
     "lint:natspec": "npx @defi-wonderland/natspec-smells --config natspec-smells.config.js",
-    "lint:sol-logic": "solhint -c .solhint.json 'src/**/*.sol' 'script/**/*.sol'",
-    "lint:sol-tests": "solhint -c .solhint.tests.json 'test/**/*.sol'",
+    "lint:sol": "solhint 'src/**/*.sol' 'test/**/*.sol' 'script/**/*.sol'",
```

Comment:
no point in `:sol` now no?

Reply:
not really lol. Do you think it makes sense to keep separate scripts to lint logic and tests?

Reply:
not really :/

Reply:
changed it to just `lint`